### PR TITLE
4.9.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 4.9.0 (2021-04-15)
+
+- Updated the stackblitz demos to use Angular 9. [#160](https://github.com/blackbaud/skyux-docs-tools/pull/160)
+- Fixed the stackblitz demos to properly show font awesome and skyux icons. [#160](https://github.com/blackbaud/skyux-docs-tools/pull/160)
```
Comment (blackbaud-paulah) [inline CHANGELOG.md:4]:
```suggestion
- Fixed the stackblitz demos to properly show font awesome and SKY UX icons. [#160](https://github.com/blackbaud/skyux-docs-tools/pull/160)
```

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-4.9.0-release](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-4.9.0-release)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# 4.9.0 (2021-04-15)
+
+- Updated the stackblitz demos to use Angular 9. [#160](https://github.com/blackbaud/skyux-docs-tools/pull/160)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]:
```suggestion
- Updated the StackBlitz demos to use Angular 9. [#160](https://github.com/blackbaud/skyux-docs-tools/pull/160)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
+- Fixed the stackblitz demos to properly show font awesome and SKY UX icons. [#160](https://github.com/blackbaud/skyux-docs-tools/pull/160)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:4]:
```suggestion
- Fixed the StackBlitz demos to properly display Font Awesome and SKY UX icons. [#160](https://github.com/blackbaud/skyux-docs-tools/pull/160)
```

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-4.9.0-release](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-4.9.0-release)

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-4.9.0-release](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-4.9.0-release)

**Review by blackbaud-paulah — APPROVED** [review]

**Pull request merged**